### DISCUSSION TITLE
Beginner

### DISCUSSION THREAD

**Discussion by force2brnkdwith2785** [community/community#65957] [New to GitHub]
### Select Topic Area

General

### Body

I'm new at this but this is confusing, and trying to learn it . Wish they were away you could help someone new .. I wanna root phones but love to show how and what apps u need

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @force2brnkdwith2785  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
